### DISCUSSION TITLE
Ps2 Version

### DISCUSSION THREAD

**Discussion by HudsonSchumaker** [diasurgical/DevilutionX#5795] [Q&A] [answered]
Hi
Are you using SDL2 for ps2 ? 

regards.

**Comment by AJenbo** [comment]
Yes https://github.com/diasurgical/devilutionX/pull/5238

Except for sound which is using the ps2sdk audsrv directly.

**Reply by HudsonSchumaker** [reply]
> Yes https://github.com/diasurgical/devilutionX/pull/5238
> 
> 
> 
> Except for sound which is using the ps2sdk audsrv directly.

It was difficult to set up ? 
I am assuming you are also using ps2dev. I created a bare minimum example, I can clear the background, but the texture is not being render. So my question is the type of the texture like ARGB or RGBA etc...

Thanks in advance

**Reply by AJenbo** [reply]
It was pretty easy to get going.

Use SDL_PIXELFORMAT_ABGR1555 or you will get performance issues. You can go though our code to see how it works.

**Reply by HudsonSchumaker** [reply]
interesting, i will take a look, thanks a lot

**Reply by HudsonSchumaker** [reply]
could please give me a hint where to look ?
regards

**Reply by AJenbo** [reply, marked as answer]
https://github.com/diasurgical/devilutionX/blob/master/Source/utils/display.cpp#L241
https://github.com/diasurgical/devilutionX/blob/9b65eca49b5b6a9640ad44cb72bd95f6e21587cf/Source/engine/dx.cpp#L214
